google-cloud-tools-plugin/google-account/resources/META-INF/google-account.xml

```diff
@@ -0,0 +1,77 @@
+<!--
+  ~ Copyright 2017 Google Inc. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<idea-plugin>
+  <application-components>
+    <component>
+      <implementation-class>
+        com.google.cloud.tools.intellij.startup.AccountPluginInitializationComponent
+      </implementation-class>
+    </component>
+  </application-components>
+
+  <project-components>
+  </project-components>
+
+  <actions>
+    <action id="GoogleLogin.LoginService"
```

Comment:
I would have thought using the same ID here as in the old account plugin would have causes exceptions on startup if the user chooses not to immediately restart (since the login widget isn't deregistered).

Reply:
yeah funnily enough it doesn't.  I didn't change it because for some reason the id had an effect on layout.  I may be crazy but JB may have whitelisted us for that right aligned spot on the toolbar.

Reply:
i tested this out, and changing the ID had no effect. 

But if there are no exceptions then we can leave it. Presumably if the user manually set keyboard shortcuts on the old login action, they would lose the shortcut if we rename the ID (not that I think anyone actually did this).